src/EA.Weee.Web/Areas/AATFReturn/Views/SentOnRemoveSite/Index.cshtml

```diff
@@ -43,32 +46,44 @@
                         B2C
                     </div>
                     <div class="govuk-grid-column-one-half weee-tonnage-column-padding-left" style="font-weight: normal">
-                        @Model.Tonnages.B2C
+                        @Html.HiddenFor(m => m.TonnageB2C)
+                        @Model.TonnageB2C
                     </div>
                 </div>
                 <div class="govuk-grid-row govuk-table__row govuk-table__header" style="width: 50%">
                     <div class="govuk-grid-column-one-half">
                         B2B
                     </div>
                     <div class="govuk-grid-column-one-half weee-tonnage-column-padding-left" style="font-weight: normal">
-                        @Model.Tonnages.B2B
+                        @Html.HiddenFor(m => m.TonnageB2B)
+                        @Model.TonnageB2B
                     </div>
                 </div>
             </div>
         </div>
-
-        <div class="govuk-grid-column-full govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedValue)" style="padding-left: 0px; margin-bottom: 10px;">
-            @this.WeeeGds().RadioButtonsFor(
-                m => m.SelectedValue,
-                Model.PossibleValues,
-                "Confirm if you want to remove this AATF/ATF from your list",
-                RadioButtonLegend.VisuallyHidden,
-                RadioButtonLayout.Inline)
+        <div class="govuk-form-group" style="padding-bottom: 10px">
+            <div class="govuk-grid-column-full govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedValue)" style="padding-left: 0px; margin-bottom: 10px;">
+                @this.WeeeGds().RadioButtonsFor(
+                    m => m.SelectedValue,
+                    Model.PossibleValues,
+                    "Confirm if you want to remove this AATF/ATF from your list",
+                    RadioButtonLegend.VisuallyHidden,
+                    RadioButtonLayout.Inline)
+            </div>
+            <img src="~/Content/govuk_frontend/images/icon-important.png" alt="icon-important" style="float:left;" height="42" width="42">
+            <p class="important">
+                Removing this AATF/ATF will delete all tonnage data associated with it.
+            </p>
         </div>
-        <p class="govuk-body-s">Removing this AATF/ATF will delete all tonnage data associated with it.</p>
+
         <button type="submit" name="save" class="govuk-button" id="site-details">
             Save and continue
         </button>
     }
 
-</section>
\ No newline at end of file
+</section>
+<style type="text/css">
```

Comment:
can this be put is an a scss file?  I notice you have the same styling section in the other PR for Reused remove site